src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -130,6 +125,12 @@ private void checkPosition(int index) {
         }
     }
 
+    private void checkPositionAdd(int index) {
+        if (index < -1 || index >= size) {
```

Comment:
now you're allowing a -1 index. also, from previous example this check still does not allow to place on index 10 with size 10, because if index = 10 - condition `index >= size` will be true and an exception will be thrown

Reply:
It isn't so.
Because I gave in the method (index -1), but I rewrote it.